code/modules/uplink/uplink_item.dm

```diff
@@ -110,10 +110,10 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	var/obj/item/weapon/storage/box/B = A
 	if(istype(B) && B.contents.len > 0)
 		for(var/obj/item/I in B)
-			U.purchase_log += "<big>[bicon(I)]</big>"
+			U.purchase_log += "<big>[icon2base64html(I)]</big>"
 	else
 		if(purchase_log_vis)
-			U.purchase_log += "<big>[bicon(A)]</big>"
+			U.purchase_log += "<big>[icon2base64html(A)]</big>"
```

Comment:
These are intentionally kept as the base64 version because other wise we would have to send the icon to all users, but this is generated at purchase, so doing it that way would give clients the ability to know what tators have purchased by looking at their cache folder for new files during the round, and that kind of meta leak would be bad.